### ISSUE TITLE
React.createClass + React.PropTypes warnings

### ISSUE THREAD

**Issue by zslabs** [FormidableLabs/component-playground#92]
With the [15.5.0 release](https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html), React is logging warnings about using the two items mentioned in the title. There is a [react-codemod](https://github.com/reactjs/react-codemod) available to migrate this -- but wasn't sure if you had anything else going on in the codebase dependent on older React versions.  Thanks!

**Comment by kylecesmat** [comment]
This is partially blocked by https://github.com/JedWatson/react-codemirror/pull/97 right now.

Once this dependency is updated it should be an easy fix to convert `createClass` and import `PropTypes` from the proper package and get rid of these warnings on 15.5+ 👍

**Comment by zslabs** [comment]
Hi,
Just a heads up, looks like that's been merged now.

**Comment by kylecesmat** [comment]
@zslabs thanks! We'll work on getting #93 merged followed by a new PR with react-codemirror@1.0

**Comment by zslabs** [comment]
Thanks for getting this in!
